Pipelines/ci.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# CI build producing developer packages.
+
+variables:
+- template: Config/settings.yaml
+- name: Version.Revision
+  value: $[counter(variables['MRTKGraphicsToolsVersion'], 1)]
+- name: Version.PreReleaseRevision
+  value: $[counter(variables['MRTKGraphicsToolsReleaseTag'], 1)]
+
+resources:
+  repositories:
+  - repository: PipelineTools
+    type: git
+    endpoint: ToolsAccess
+    name: tools.internal
+    ref: 78ecf591964ecf511b4c7a0cd84d98ce11e8be16
+
+pr: none # disable PR validation
+trigger:
+  tags:
+    include:
+    - '*'
+  branches:
+    exclude:
+    - main
+    - user-tayomadein-ci-cd
+
+jobs:
+- job: UnityValidation
+  pool: Unity_2021.3.3f1_Pool
+  steps:
+  - checkout: self
+  
+  - checkout: PipelineTools
+
+  - pwsh: Write-Host "##vso[build.updatebuildnumber]$(MRTKGraphicsToolsVersion)"
```

Comment:
Will `$(MRTKGraphicsToolsVersion)` change for each run? If not, you might want to update this to make the build number unique. If it will, then no problem!

Reply:
The current plan is to run for tags, in that case, $(MRTKGraphicsToolsVersion) will change for each run.